pkg/webhook/bootstrap.go

```diff
@@ -285,19 +306,22 @@ func (s *Server) mutatingWHConfigs() (runtime.Object, error) {
 		if err != nil {
 			return nil, err
 		}
+		cc, err := s.clientConfigWithPath(path)
+		if err != nil {
+			return nil, err
+		}
+		wh.ClientConfig = *cc
 		mutatingWebhooks = append(mutatingWebhooks, *wh)
-
 	}
 
 	if len(mutatingWebhooks) > 0 {
 		return &admissionregistration.MutatingWebhookConfiguration{
 			TypeMeta: metav1.TypeMeta{
-				APIVersion: "admissionregistration.k8s.io/v1beta1",
+				APIVersion: fmt.Sprintf("%s/%s", admissionregistration.GroupName, "v1beta1"),
```

Comment:
I think `http` pkg has some utility methods to manipulate the URL paths, would suggest using those.

Reply:
Hmm... Why handling apiVersion with URL manipulation util in http pkg?